src/index.tsx

```diff
@@ -13,34 +13,44 @@ interface SpoilerProps {
 export const Spoiler: React.FunctionComponent<SpoilerProps> = ({
   text = null,
   textColor = 'inherit',
-  hiddenColor = '#444',
-  revealedColor = '#e2e2e2',
+  hiddenColor = 'currentColor',
+  revealedColor = 'transparent',
```

Comment:
`currentColor` will always be the text color and I would guess in 99% of cases, you would want your text to be "regular" after the reveal.